### DISCUSSION TITLE
Bug: Can't add a shipping service because I removed the pickup option

### DISCUSSION THREAD

**Discussion by studiodrimmel** [swellstores/community#450] [Help] [answered]
I have removed the shipping services and I also removed the local pickup option. That results in a blank page since pickup is not defined anymore:

![Scherm­afbeelding 2023-11-30 om 15 38 37](https://github.com/swellstores/community/assets/101345863/77394e07-6d41-4785-8a29-eee77d8e2810)
![Scherm­afbeelding 2023-11-30 om 15 33 49](https://github.com/swellstores/community/assets/101345863/5739e0f6-e0a7-4dd8-b791-02351591ec8f)
![Scherm­afbeelding 2023-11-30 om 15 36 37](https://github.com/swellstores/community/assets/101345863/bcacaf2c-8e6c-41fc-8565-c949bdf7848d)

(ps: the api's are currently slow as hell)

**Comment by ericingram** [comment]
Will have a fix for that up shortly

**Reply by ericingram** [reply, marked as answer]
Should be fixed now
